src/Sylius/Behat/Context/Ui/Shop/CartContext.php

```diff
@@ -527,6 +547,22 @@ public function iShouldSeeCartTotalQuantity(int $totalQuantity): void
         Assert::same($this->cartWidgetElement->getCartTotalQuantity(), $totalQuantity);
     }
 
+    /**
+     * @Then I should be on the checkout page
```

Comment:
Couldn't you just reuse the `I should be on the checkout addressing step` from `CheckoutAddressingContext` instead of injecting the address page in here?

Reply:
After a call with @GSadee we've decided we left it as is.

Reply:
To clarify, there is a duplicated step, so we can't use the step from `CheckoutAddressingContext`. We've decided it's not worth (at least in this particular task) to refactor this and we left it as is.